src/Components/Common/CommonComponents/PopUp.jsx

```diff
@@ -61,14 +49,6 @@ const PopUp = ({
                     <option key={option.value} value={option.value}>{option.label}</option>
                   ))}
                 </select>
-              ) : field.type === 'file' ? (
-                <input
-                  type="file"
-                  id={field.id}
-                  name={field.id}
-                  onChange={handleFileChange}
-                  className="border border-gray-300 rounded-md shadow-sm mt-1 block w-full cursor-pointer bg-blue-500 text-white px-4 py-2 hover:bg-blue-600"
-                />
```

Comment:
why did you remove the above?^^ will this impact anything or did we rewrite the logic anywhere else.